### PR TITLE
Add reg files to set Files as default

### PR DESCRIPTION
**Pull request by gave92**
@yaichenbaum Since this topic is all the rage this PR tries to make setting Files as default a little easier and safer.
- Solves the issue of not being able to open the control panel
- Moves registry changes from LOCAL_MACHINE to CURRENT_USER (only affects the current user)
- Adds .reg files to automatically set and unset Files as default
- The .reg file also sets Win+E to open Files

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@gave92 Should we merge the two doc pages?

**Comment by gave92** [comment]
Maybe not, there might be people willing to set win+e to open files but not replace explorer entirely

**Comment by yaichenbaum** [comment]
@gave92 I was referring to putting all the info on the same page, I still think the scripts and instructions should be kept separate. I would also like to keep the instructions for the autohotkey script as an alternative to editing the registry.

**Comment by gave92** [comment]
You mean merging the "win+e" page with the "set files as default page"?
I don't mind them being separate, too much difficult info on one page :)

And yes we should list both methods for win+e, registry or autohotkey.

**Comment by yaichenbaum** [comment]
@gave92 In that case, would you mind moving the script for `win` + `e` to the correct page?

**Comment by gave92** [comment]
You know? I've set Files as default myself and it's not bad at all xD

File: docs/articles/configure-win-e.md [hunk 1]
```diff
@@ -15,4 +18,14 @@ FilesLocation := USERPROFILE . "\AppData\Local\Microsoft\WindowsApps\files.exe"
...
+
+**Option 2: modifying the registry**
+
+*This method involves modifying the registry, make sure to create a backup beforehand and proceed at your own risk. Please keep in mind that this method is unsupported and may not work for everyone.*
+
+You can also configure Files to launch using the Win + E shortcut without an external program:
+1. Create a backup of the registry, make sure to store the backup in your desktop folder so that you can access it in the event that Files won't open.
+2. Download *both* these .reg files [1](https://github.com/files-community/files-community.github.io/data/UndoOpenFilesOnWinE.reg) and [2](https://github.com/files-community/files-community.github.io/data/OpenFilesOnWinE.reg) and place them *on the desktop*
+3. Run `OpenFilesOnWinE.reg` to open Files on Win+E
+4. Run `UndoOpenFilesOnWinE.reg` to restore windows explorer
```
Comment (yaichenbaum) [inline docs/articles/configure-win-e.md:31]: Should we add instructions for manually editing the registry?

Comment (gave92) [reply docs/articles/configure-win-e.md:33]: Done 👍

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@gave92 Great work! LGTM

**Comment by gave92** [comment]
Thanks! Have you tried the reg file? It's tested on my on only :)

**Comment by yaichenbaum** [comment]
@gave92 I just realized the download links don't seem to be working. I will try them out once the download link is fixed 😆

**Comment by gave92** [comment]
Do you know what kind of url is needed. The url that starts with "raw.github." does not start a download

**Comment by yaichenbaum** [comment]
> Do you know what kind of url is needed? The urls that starts with "raw.github." do not start a download but only show the file they point to
> The files are in the "main/data" folder

I think it just needs `<a href="">Download</>`.

**Comment by gave92** [comment]
Still opens the file :/ maybe because it's a text file. I can try putting the 2 scripts in a zip maybe

**Pull request merged**